ingress/controllers/nginx-third-party/nginx/main.go

```diff
@@ -220,9 +220,9 @@ type NginxManager struct {
 	// path to the configuration file to be used by nginx
 	ConfigFile string
 
-	sslCertificates []ssl.Certificate
-	sslDHParam      string
-	servicesL4      []Service
+	sslDHParam string
```

Comment:
Please add a comment describing high level function of NginxManager and a oneline about sslDHParam.